Hive.NET.Core/Extensions/Extensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using Hive.NET.Core.Api;
+using Hive.NET.Core.Components;
+using Hive.NET.Core.Models.Enums;
+
+namespace Hive.NET.Core.Extensions;
+
+public static class Extensions
+{
+    internal static Components.Hive MapFromDetails(this HiveDetailsDto detailsDto) =>
+        new()
+        {
+            Id = detailsDto.Id,
+            _name = detailsDto.Name,
+            Swarm = detailsDto.Swarm.Select(x => new Bee()
+            {
+                IsWorking = x.IsWorking,
+                Id = x.Id
+            }).ToList(),
+            Items = detailsDto.WorkItems.ToList().Select(workItem =>
```

Comment:
ToList() seems unnecessary. WorkItems is already a list

Reply:
done